controller/workitem.go

```diff
@@ -605,6 +611,26 @@ func ConvertWorkItem(request *goa.RequestData, wi workitem.WorkItem, additional
 	return op
 }
 
+// WorkItemIncludeHasChildren adds meta information about existing children
+func WorkItemIncludeHasChildren(appl application.Application, ctx context.Context) WorkItemConvertFunc {
+	// TODO: Wrap ctx in a Timeout context?
+	return func(request *goa.RequestData, wi *workitem.WorkItem, wi2 *app.WorkItem) {
+		hasChildren, err := appl.WorkItemLinks().WorkItemHasChildren(ctx, wi.ID)
+		if err != nil {
```

Comment:
It looks strange to me that you just log the error but continue with the flow. Shouldn't you enforce the `hasChildren` element value to `false` when an error occurred ?